packages/dapp/src/const/MenuConfig/useWalletSelectContent.tsx

```diff
@@ -34,43 +50,62 @@ export const useWalletSelectContent = () => {
     [connect, onWalletConnect],
   );
 
+  useEffect(() => {
+    verifyMultisigEnvironment();
+  }, [account?.address]);
+
   const walletMenuItems = useMemo<MenuListItem[]>(() => {
-    const installedWallets = supportedWallets.filter((wallet) =>
-      wallet.installed(),
+    const installedWallets = supportedWallets.filter(
+      async (wallet) => await wallet.installed(),
```

Comment:
Async `filter` doesn't work like that.  Please review this line in the debugger and see how it works 🙂 Also, check `SelectWalletPage` implementation in the widget.